cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_internal_motion.py

```diff
@@ -146,17 +147,15 @@ def init_pos():
             # save data
             for data_kraken in data_krakens:
                 data_kraken.writeAllData()
-        except rospy.ROSInterruptException as e:
```

Comment:
This is gone now, while the general exception is handled a bit different. Functionally will be the same though